andela/travel_tool_back
# #160820700 Email notification for new travel requests

#### What does this PR do?
* This PR sends email notifications for new travel requests
#### Description of Task to be completed?
* Create a email notification engine using [sendgrid](https://www.npmjs.com/package/@sendgrid/mail) api
* Create email template
* Send email on successfully creating new request
#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- Rollback and run database migrations - `yarn db:rollmigrate`
- Install dependencies and start the server - `make start`
-  [Start](https://github.com/andela/travel_tool_front/pull/107)  the frontend.
- Make a request and choose yourself as the manager
- Check your gmail account for a new message from travela regarding your new request
- Click on the `view request` button to redirect to the request
#### Any background context you want to provide?
- In other to test this feature, you need to update your role on the Users table to that of a manager.
Manager roleId: `50319`

#### What are the relevant pivotal tracker stories?
* [#160820700](https://www.pivotaltracker.com/story/show/160820700)

#### Screenshots (if appropriate)
* N/A
#### Questions:
* N/A
